docs/technology_stack.adoc

```diff
@@ -7,35 +7,42 @@
 toc::[]
 
 == Overview
-The following sections describe the technology stack used by Planner.
+The Planner is an Angular app built on top of Node.js. It can run as a standalone app, or be used as a library to be integrated into the link:./fabric8-platform[Fabric8 Platform].
 
-== Angular 4
-<TBD>
+Development or deployment of Planner is fairly platform agnostic, so we can skip the operating system requirements and discuss from the runtime environment to way-up.
 
-== CSS and SASS
-<WIP, to be changed to PF CSS, Less>
+== Development Runtime
+Planner uses *`Node.js`* as the platform-runtime, on top of which it builds; on that line, we use *`npm`* for resolving and distributing packages for the project.
```

Comment:
Actually here the platform-runtime is completely different from what we generally term the platform (Fabric8-UI) or runtime (the minimal frontend for standalone Planner). In this case platoform runtime is a piece of software that enables a language to interface with the OS (in this case the language is JS and the platform-runtime is Node.js). Sorry for the confusion, but there's barely any way around as those are the apt terms here.

`*uses Node.js* as the platform-runtime` and `uses Node.js for building the platform-runtime` are completely different things, and the later isn't intended here (we're not building Node.js, we're using it for making JS interact with the platform, which can be Linus, Windows, OSX etc.)

Also, the packages resolved are a long list and is already available inside `package.json` (standard process for Node,js apps), don't think we need to list hundreds of package names here that's prone to changes often.